sw/airborne/Makefile

```diff
@@ -38,16 +38,16 @@ VPATH = .
 ifneq ($(MAKECMDGOALS),clean)
   include $(AIRCRAFT_BUILD_DIR)/Makefile.ac
   $(TARGET).srcs += $($(TARGET).EXTRA_SRCS)
-  include ../../conf/Makefile.local
+  include $(PAPARAZZI_HOME)/conf/Makefile.local
```

Comment:
yes, it was more about all the makefiles in conf, like the ones below.
with the relative path (as it was before) will "automatically" look for them in SRC